### PR TITLE
Fix prose in Abstract section

### PR DESCRIPTION
**Pull request by pozdnyakov**

### CODE REVIEW

File: index.bs [hunk 1]
```diff
@@ -11,8 +11,7 @@ Editor: Kenneth Rohde Christiansen      , Intel Corporation, http://intel.com/
 Editor: Anssi Kostiainen 41974, Intel Corporation, http://intel.com/
 Group: dap
 Abstract:
-  This specification defines a concrete sensor interface to monitor
-  the rate of rotation around the device's local three primary axes.
+  This specification defines concrete sensor interfaces to monitor the device's physical orientation.
```
Comment (kenchris) [inline index.bs]: Should it say in reference to what?

Comment (pozdnyakov) [inline index.bs]: Yes, done.

**Comment by pozdnyakov** [comment]
@kenchris PTAL

**Comment by kenchris** [comment]
lgtm

**Pull request merged**